machines.2
NAME		TYPE			CPU	MEM	OS		USE	LOCATION	S/N		MAC ADDRESS
----		----			---	---	--		---	--------	---		-----------
albany		HP-Z420-XeonE5-2650v2	8x2.6G	32Gb	Linux(Fedora)	general	120-unix-lab	2UA349121X	2c:44:fd:26:90:0e
annapolis	HP-Z420-XeonE5-2650v2	8x2.6G	32Gb	Linux(Fedora)	general	120-unix-lab	2UA349121Q	2c:44:fd:26:8f:f0
atlanta		HP-Z420-XeonE5-2650v2	8x2.6G	32Gb	Linux(Fedora)	general	120-unix-lab	2UA349120Y	2c:44:fd:15:3b:a7
augusta		HP-Z420-XeonE5-2650v2	8x2.6G	32Gb	Linux(Fedora)	general	120-unix-lab	2UA349120H	2c:44:fd:26:8c:81
austin		HP-Z420-XeonE5-2650v2	8x2.6G	32Gb	Linux(Fedora)	general	120-unix-lab	2UA349121W	2c:44:fd:15:3d:ca
baton-rouge	HP-Z420-XeonE5-2650v2	8x2.6G	32Gb	Linux(Fedora)	general	120-unix-lab	2UA349120T	2c:44:fd:1c:7b:bd
bismarck	HP-Z420-XeonE5-2650v2	8x2.6G	32Gb	Linux(Fedora)	general	120-unix-lab	2UA349121G	2c:44:fd:15:3d:cd
boise		HP-Z420-XeonE5-2650v2	8x2.6G	32Gb	Linux(Fedora)	general	120-unix-lab	2UA349120V	74:46:a0:c3:bd:c6
boston		HP-Z420-XeonE5-2650v2	8x2.6G	32Gb	Linux(Fedora)	general	120-unix-lab	2UA349121N	2c:44:fd:15:3d:c7
carson-city	HP-Z420-XeonE5-2650v2	8x2.6G	32Gb	Linux(Fedora)	general	120-unix-lab	2UA349120X	2c:44:fd:26:8f:6c
charleston	HP-Z420-XeonE5-2650v2	8x2.6G	32Gb	Linux(Fedora)	general	120-unix-lab	2UA349120P	2c:44:fd:1c:79:81
cheyenne	HP-Z420-XeonE5-2650v2	8x2.6G	32Gb	Linux(Fedora)	general	120-unix-lab	2UA3491211	74:46:a0:c3:bd:7e
columbia	HP-Z420-XeonE5-2650v2	8x2.6G	32Gb	Linux(Fedora)	general	120-unix-lab	2UA349121S	74:46:a0:c3:be:ba
columbus-oh	HP-Z420-XeonE5-2650v2	8x2.6G	32Gb	Linux(Fedora)	general	120-unix-lab	2UA349120Z	2c:44:fd:26:8e:3b
concord		HP-Z420-XeonE5-2650v2	8x2.6G	32Gb	Linux(Fedora)	general	120-unix-lab	2UA349120Q	2c:44:fd:15:3c:1d
denver		HP-Z420-XeonE5-2650v2	8x2.6G	32Gb	Linux(Fedora)	general	120-unix-lab	2UA349121Y	74:46:a0:c3:be:b9
des-moines	HP-Z420-XeonE5-2650v2	8x2.6G	32Gb	Linux(Fedora)	general	120-unix-lab	2UA349121L	2c:44:fd:15:3d:c5
dover		HP-Z420-XeonE5-2650v2	8x2.6G	32Gb	Linux(Fedora)	general	120-unix-lab	2UA3491218	2c:44:fd:15:3b:5c
frankfort	HP-Z420-XeonE5-2650v2	8x2.6G	32Gb	Linux(Fedora)	general	120-unix-lab	2UA349120K	2c:44:fd:1c:78:57
harrisburg	HP-Z420-XeonE5-2650v2	8x2.6G	32Gb	Linux(Fedora)	general	120-unix-lab	2UA349120N	2c:44:fd:1c:79:1b
hartford	HP-Z420-XeonE5-2650v2	8x2.6G	32Gb	Linux(Fedora)	general	120-unix-lab	2UA349121B	2c:44:fd:1c:77:d6
helena		HP-Z420-XeonE5-2650v2	8x2.6G	32Gb	Linux(Fedora)	general	120-unix-lab	2UA349120S	2c:44:fd:1c:7b:26
honolulu	HP-Z420-XeonE5-2650v2	8x2.6G	32Gb	Linux(Fedora)	general	120-unix-lab	2UA349121T	2c:44:fd:1c:7c:64
indianapolis	HP-Z420-XeonE5-2650v2	8x2.6G	32Gb	Linux(Fedora)	general	120-unix-lab	2UA349120R	2c:44:fd:1c:7b:c4
jackson		HP-Z420-XeonE5-2650v2	8x2.6G	32Gb	Linux(Fedora)	general	120-unix-lab	2UA349120W	2c:44:fd:26:8f:63
jefferson-city	HP-Z420-XeonE5-2650v2	8x2.6G	32Gb	Linux(Fedora)	general	120-unix-lab	2UA3491213	2c:44:fd:26:8f:68
juneau		HP-Z420-XeonE5-2650v2	8x2.6G	32Gb	Linux(Fedora)	general	120-unix-lab	2UA349121H	2c:44:fd:15:3d:d1
lansing		HP-Z420-XeonE5-2650v2	8x2.6G	32Gb	Linux(Fedora)	general	120-unix-lab	2UA349121C	2c:44:fd:15:3b:58
lincoln		HP-Z420-XeonE5-2650v2	8x2.6G	32Gb	Linux(Fedora)	general	120-unix-lab	2UA3491215	2c:44:fd:26:8f:6b
little-rock	HP-Z420-XeonE5-2650v2	8x2.6G	32Gb	Linux(Fedora)	general	120-unix-lab	2UA349121J	2c:44:fd:1c:7c:61
madison		HP-Z420-XeonE5-2650v2	8x2.6G	32Gb	Linux(Fedora)	general	120-unix-lab	2UA3491212	2c:44:fd:1c:7a:71
montgomery	HP-Z420-XeonE5-2650v2	8x2.6G	32Gb	Linux(Fedora)	general	120-unix-lab	2UA3491212	2c:44:fd:15:3d:c6
montpelier	HP-Z420-XeonE5-2650v2	8x2.6G	32Gb	Linux(Fedora)	general	120-unix-lab	2UA349120M	2c:44:fd:1c:78:f0
nashville	HP-Z420-XeonE5-2650v2	8x2.6G	32Gb	Linux(Fedora)	general	120-unix-lab	2UA349120J	2c:44:fd:26:8c:82
oklahoma-city	HP-Z420-XeonE5-2650v2	8x2.6G	32Gb	Linux(Fedora)	general	120-unix-lab	2UA3491221	2c:44:fd:1c:7a:74
olympia		HP-Z420-XeonE5-2650v2	8x2.6G	32Gb	Linux(Fedora)	general	120-unix-lab	2UA3491216	2c:44:fd:15:3b:0d
phoenix		HP-Z420-XeonE5-2650v2	8x2.6G	32Gb	Linux(Fedora)	general	120-unix-lab	2UA3491219	2c:44:fd:26:8d:1d
pierre		HP-Z420-XeonE5-2650v2	8x2.6G	32Gb	Linux(Fedora)	general	120-unix-lab	2UA349121D	2c:44:fd:26:8c:b1
providence	HP-Z420-XeonE5-2650v2	8x2.6G	32Gb	Linux(Fedora)	general	120-unix-lab	2UA349121R	2c:44:fd:26:90:0f
raleigh		HP-Z420-XeonE5-2650v2	8x2.6G	32Gb	Linux(Fedora)	general	120-unix-lab	2UA3491214	74:46:a0:c3:bd:9e
richmond	HP-Z420-XeonE5-2650v2	8x2.6G	32Gb	Linux(Fedora)	general	120-unix-lab	2UA349121K	2c:44:fd:26:90:10
sacramento	HP-Z420-XeonE5-2650v2	8x2.6G	32Gb	Linux(Fedora)	general	120-unix-lab	2UA349121Z	2c:44:fd:15:3d:cb
saint-paul	HP-Z420-XeonE5-2650v2	8x2.6G	32Gb	Linux(Fedora)	general	120-unix-lab	2UA3491210	2c:44:fd:1c:7a:21
salem		HP-Z420-XeonE5-2650v2	8x2.6G	32Gb	Linux(Fedora)	general	120-unix-lab	2UA349121P	74:46:a0:c3:be:b3
salt-lake-city	HP-Z420-XeonE5-2650v2	8x2.6G	32Gb	Linux(Fedora)	general	120-unix-lab	2UA3491217	2c:44:fd:15:3a:c2
santa-fe	HP-Z420-XeonE5-2650v2	8x2.6G	32Gb	Linux(Fedora)	general	120-unix-lab	2UA3491220	74:46:a0:c3:be:8e
springfield	HP-Z420-XeonE5-2650v2	8x2.6G	32Gb	Linux(Fedora)	general	120-unix-lab	2UA349121V	2c:44:fd:1c:7c:5f
tallahassee	HP-Z420-XeonE5-2650v2	8x2.6G	32Gb	Linux(Fedora)	general	120-unix-lab	2UA349120G	2c:44:fd:1c:77:e6
topeka		HP-Z420-XeonE5-2650v2	8x2.6G	32Gb	Linux(Fedora)	general	120A-unix-lab	2UA349120L	2c:44:fd:26:8d:21
trenton		HP-Z420-XeonE5-2650v2	8x2.6G	32Gb	Linux(Fedora)	general	120A-unix-lab	2UA349121F	2c:44:fd:15:3a:c9
bentley		HP-Z440-XeonE5-1650v3	6x3.5Gh	32Gb	Linux(Fedora)	general	325-hpc-lab	2UA5351R0X	ec:b1:d7:4c:a0:c1
bugatti		HP-Z440-XeonE5-1650v3	6x3.5Gh	32Gb	Linux(Fedora)	general	325-hpc-lab	2UA5351R0Y	ec:b1:d7:50:4d:11
ferrari		HP-Z440-XeonE5-1650v3	6x3.5Gh	32Gb	Linux(Fedora)	general	325-hpc-lab	2UA5351R10	ec:b1:d7:56:01:7a
jaguar		HP-Z440-XeonE5-1650v3	6x3.5Gh	32Gb	Linux(Fedora)	general	325-hpc-lab	2UA5351R0V	ec:b1:d7:50:4e:a2
lamborghini	HP-Z440-XeonE5-1650v3	6x3.5Gh	32Gb	Linux(Fedora)	general	325-hpc-lab	2UA5351R0W	ec:b1:d7:56:02:ae
lotus		HP-Z440-XeonE5-1650v3	6x3.5Gh	32Gb	Linux(Fedora)	general	325-hpc-lab	2UA5351R11	ec:b1:d7:4c:a0:c0
maserati	HP-Z440-XeonE5-1650v3	6x3.5Gh	32Gb	Linux(Fedora)	general	325-hpc-lab	2UA5351R0Z	ec:b1:d7:50:4d:10
porsche		HP-Z440-XeonE5-1650v3	6x3.5Gh	32Gb	Linux(Fedora)	general	325-hpc-lab	2UA5351R0T	ec:b1:d7:4c:a1:a7
corvette	HP-Z440-XeonE5-1650v3	6x3.5Gh	32Gb	Linux(Fedora)	general	475-machine-rm	2UA54518CT	ec:b1:d7:59:3f:9d
mustang		HP-Z440-XeonE5-1650v3	6x3.5Gh	32Gb	Linux(Fedora)	general	475-machine-rm	2UA54518CV	50:65:f3:2a:40:1f
ankara		HP-Z230-XeonE3-1231v3	4x3.4G	32Gb	Linux(Fedora)	general	215-interac-lab	2UA51328KH	8c:dc:d4:29:ab:ec
baghdad		HP-Z230-XeonE3-1231v3	4x3.4G	32Gb	Linux(Fedora)	general	215-interac-lab	2UA51328JS	8c:dc:d4:2b:df:02
bangkok		HP-Z230-XeonE3-1231v3	4x3.4G	32Gb	Linux(Fedora)	general	215-interac-lab	2UA51328KL	8c:dc:d4:27:f1:f2
beijing		HP-Z230-XeonE3-1231v3	4x3.4G	32Gb	Linux(Fedora)	general	215-interac-lab	2UA51328KK	8c:dc:d4:29:ab:f3
berlin		HP-Z230-XeonE3-1231v3	4x3.4G	32Gb	Linux(Fedora)	general	215-interac-lab	2UA51328K6	8c:dc:d4:2b:e0:58
bogota		HP-Z230-XeonE3-1231v3	4x3.4G	32Gb	Linux(Fedora)	general	215-interac-lab	2UA51328K3 	8c:dc:d4:27:f0:3b
cairo		HP-Z230-XeonE3-1231v3	4x3.4G	32Gb	Linux(Fedora)	general 215-interac-lab	2UA51328JP	8c:dc:d4:2b:df:00
damascus	HP-Z230-XeonE3-1231v3	4x3.4G	32Gb	Linux(Fedora)	general	215-interac-lab	2UA51328JZ	8c:dc:d4:2b:df:09
dhaka		HP-Z230-XeonE3-1231v3	4x3.4G	32Gb	Linux(Fedora)	general	215-interac-lab	2UA51328KG	8c:dc:d4:27:f1:f9
hanoi		HP-Z230-XeonE3-1231v3	4x3.4G	32Gb	Linux(Fedora)	general	215-interac-lab	2UA51328K7	8c:dc:d4:29:ab:e2
hong-kong	HP-Z230-XeonE3-1231v3	4x3.4G	32Gb	Linux(Fedora)	general	215-interac-lab	2UA51328JV	8c:dc:d4:29:aa:d8
jakarta		HP-Z230-XeonE3-1231v3	4x3.4G	32Gb	Linux(Fedora)	general	215-interac-lab	2UA51328JY	8c:dc:d4:2b:df:4a
kabul		HP-Z230-XeonE3-1231v3	4x3.4G	32Gb	Linux(Fedora)	general	215-interac-lab	2UA51328KJ	8c:dc:d4:29:ab:e5
kinshasa	HP-Z230-XeonE3-1231v3	4x3.4G	32Gb	Linux(Fedora)	general	215-interac-lab	2UA51328K5	8c:dc:d4:27:f1:e7
lima		HP-Z230-XeonE3-1231v3	4x3.4G	32Gb	Linux(Fedora)	general	215-interac-lab	2UA51328K8	8c:dc:d4:27:f1:f4
london		HP-Z230-XeonE3-1231v3	4x3.4G	32Gb	Linux(Fedora)	general	215-interac-lab	2UA51328JW	8c:dc:d4:27:f0:73
madrid		HP-Z230-XeonE3-1231v3	4x3.4G	32Gb	Linux(Fedora)	general	215-interac-lab	2UA51328KF	8c:dc:d4:27:f1:ff
mexico-city	HP-Z230-XeonE3-1231v3	4x3.4G	32Gb	Linux(Fedora)	general	215-interac-lab	2UA51328K4	8c:dc:d4:27:f1:eb
moscow		HP-Z230-XeonE3-1231v3	4x3.4G	32Gb	Linux(Fedora)	general	215-interac-lab	2UA51328JQ	8c:dc:d4:29:ab:00
pyongyang	HP-Z230-XeonE3-1231v3	4x3.4G	32Gb	Linux(Fedora)	general	215-interac-lab	2UA51328KB	8c:dc:d4:29:ab:f4
riyadh		HP-Z230-XeonE3-1231v3	4x3.4G	32Gb	Linux(Fedora)	general	215-interac-lab	2UA51328K1	8c:dc:d4:27:f0:74
santiago	HP-Z230-XeonE3-1231v3	4x3.4G	32Gb	Linux(Fedora)	general	215-interac-lab	2UA51328KC	8c:dc:d4:29:ab:f9
seoul		HP-Z230-XeonE3-1231v3	4x3.4G	32Gb	Linux(Fedora)	general	215-interac-lab	2UA51328K2	8c:dc:d4:29:aa:d7
singapore	HP-Z230-XeonE3-1231v3	4x3.4G	32Gb	Linux(Fedora)	general	215-interac-lab	2UA51328KD	8c:dc:d4:29:ab:f1
tehran		HP-Z230-XeonE3-1231v3	4x3.4G	32Gb	Linux(Fedora)	general	215-interac-lab	2UA51328JX	8c:dc:d4:2b:df:0f
tokyo		HP-Z230-XeonE3-1231v3	4x3.4G	32Gb	Linux(Fedora)	general	215-interac-lab	2UA51328JT	8c:dc:d4:2b:df:0a
washington-dc	HP-Z230-XeonE3-1231v3	4x3.4G	32Gb	Linux(Fedora)	general	120A-unix-lab	2UA51328K9	8c:dc:d4:2b:e0:59
oslo		HP-Z230-XeonE3-1231v3	4x3.4G	32Gb	Win7		general	250-collab-rm	2UA51328	8c:dc:d4:29:AB:E7
paris		HP-Z230-XeonE3-1231v3	4x3.4G	32Gb	Win7		general	450-collab-rm	2UA51328JR	8c:dc:d4:27:F0:6E	
rome		HP-Z230-XeonE3-1231v3	4x3.4G	32Gb	Win7		general	350-collab-rm	2UA51328KO	8c:dc:d4:27:F0:ED	
acorn		HP-Z220-XeonE3-1230	4x3.3G	8Gb	Linux(Fedora)	general	225-interac-lab	2UA2390DCK	b4:b5:2f:bf:9d:82
almond		HP-Z220-XeonE3-1230	4x3.3G	8Gb	Linux(Fedora)	general	225-interac-lab	2UA2390DBK	b4:b5:2f:bf:9d:27
beech		HP-Z220-XeonE3-1230	4x3.3G	8Gb	Linux(Fedora)	general	225-interac-lab	2UA2390DBV	b4:b5:2f:bf:9c:f6
brazil		HP-Z220-XeonE3-1230	4x3.3G	8Gb	Linux(Fedora)	general	225-interac-lab	2UA2390DCW	b4:b5:2f:bf:9d:3c
butternut	HP-Z220-XeonE3-1230	4x3.3G	8Gb	Linux(Fedora)	general	225-interac-lab	2UA2390DCT	b4:b5:2f:bf:9d:78
cashew		HP-Z220-XeonE3-1230	4x3.3G	8Gb	Linux(Fedora)	general	225-interac-lab	2UA2390DCZ	b4:b5:2f:bf:9d:84
chestnut	HP-Z220-XeonE3-1230	4x3.3G	8Gb	Linux(Fedora)	general	225-interac-lab	2UA2390DC1	b4:b5:2f:bf:9c:fc
chinquapin	HP-Z220-XeonE3-1230	4x3.3G	8Gb	Linux(Fedora)	general	225-interac-lab	2UA2390DCY	b4:b5:2f:bf:9d:7f
coconut		HP-Z220-XeonE3-1230	4x3.3G	8Gb	Linux(Fedora)	general	225-interac-lab	2UA2390DBJ	b4:b5:2f:bf:9c:f1
filbert		HP-Z220-XeonE3-1230	4x3.3G	8Gb	Linux(Fedora)	general	225-interac-lab	2UA2390DCO	b4:b5:2f:bf:9d:34
ginko		HP-Z220-XeonE3-1230	4x3.3G	8Gb	Linux(Fedora)	general	225-interac-lab	2UA2390DBR	b4:b5:2f:bf:9d:86
hazelnut	HP-Z220-XeonE3-1230	4x3.3G	8Gb	Linux(Fedora)	general	225-interac-lab	2UA2390DBL	b4:b5:2f:bf:9d:96
heartnut	HP-Z220-XeonE3-1230	4x3.3G	8Gb	Linux(Fedora)	general	225-interac-lab	2UA2390DD4	b4:b5:2f:bf:9d:8a
hickory		HP-Z220-XeonE3-1230	4x3.3G	8Gb	Linux(Fedora)	general	225-interac-lab	2UA2390DCM	b4:b5:2f:bf:9c:82
lychee		HP-Z220-XeonE3-1230	4x3.3G	8Gb	Linux(Fedora)	general	225-interac-lab	2UA2390DC3	b4:b5:2f:bf:9c:e6
macadamia	HP-Z220-XeonE3-1230	4x3.3G	8Gb	Linux(Fedora)	general	225-interac-lab	2UA2390DD1	b4:b5:2f:bf:9d:85
nangai		HP-Z220-XeonE3-1230	4x3.3G	8Gb	Linux(Fedora)	general	225-interac-lab	2UA2390DCG	b4:b5:2f:bf:9c:fd
peanut		HP-Z220-XeonE3-1230	4x3.3G	8Gb	Linux(Fedora)	general	225-interac-lab	2UA2390DCF	b4:b5:2f:bf:9d:5b
pecan		HP-Z220-XeonE3-1230	4x3.3G	8Gb	Linux(Fedora)	general	225-interac-lab	2UA2390DCQ	b4:b5:2f:bf:9d:f6
pepita		HP-Z220-XeonE3-1230	4x3.3G	8Gb	Linux(Fedora)	general	225-interac-lab	2UA2390DD6	b4:b5:2f:bf:9d:98
pili		HP-Z220-XeonE3-1230	4x3.3G	8Gb	Linux(Fedora)	general	225-interac-lab	2UA2390DBZ	b4:b5:2f:bf:9c:fa
pine		HP-Z220-XeonE3-1230	4x3.3G	8Gb	Linux(Fedora)	general	225-interac-lab	2UA2390DBX	b4:b5:2f:bf:9c:f4
pinion		HP-Z220-XeonE3-1230	4x3.3G	8Gb	Linux(Fedora)	general	225-interac-lab	2UA2390DCS	b4:b5:2f:bf:9d:61
pistachio	HP-Z220-XeonE3-1230	4x3.3G	8Gb	Linux(Fedora)	general	225-interac-lab	2UA2390DC5	b4:b5:2f:bf:9c:c4
shea		HP-Z220-XeonE3-1230	4x3.3G	8Gb	Linux(Fedora)	general	225-interac-lab	2UA2390DC9	b4:b5:2f:bf:9d:68
walnut		HP-Z220-XeonE3-1230	4x3.3G	8Gb	Linux(Fedora)	general	225-interac-lab	2UA2390DCV	b4:b5:2f:bf:9d:7c
snap		HP-Z220-XeonE3-1230	4x3.3G	8Gb	custom(Win7)	general	464-jgruiz	2UA239090D	B4:B5:2F:A9:6F:CB
bananas		HP-Z800-XeonE5645-SAS	12x2.4G	96Gb	Linux(Fedora)	general	325-hpc-lab	2UA2180XJR	e8:39:35:30:4d:ac
coconuts	HP-Z800-XeonE5645-SAS	12x2.4G	96Gb	Linux(Fedora)	general	325-hpc-lab	2UA2180XJQ	e8:39:35:30:5d:0c
lemons		HP-Z200-Xeon3450-SAS	4x2.66G	8Gb	Win7		general	325-hpc-lab	2UA0330TN0
raspberries	HP-Z200-Xeon3450-SAS	4x2.66G	8Gb	Linux(Fedora)	general	325-hpc-lab	2UA0330TMX	1c:c1:de:66:14:47
pomegranates	HP-xw4600-Core2Quad-SAS	4x2.83G	8Gb	Linux(Fedora)	r/t	475-machine-rm	2UA9510WGH	00:26:55:46:2d:00
dates		HP-Z200-Xeon3450-SAS	4x2.66G	8Gb	Linux(Fedora)	general	475-machine-rm	2UA0330TMR	1c:c1:de:66:d2:d0
brahms		HP-Z200-Xeon3450-SAS	4x2.66G	8Gb	Win7		r/t	468-ghosh	2UA0330TMY	1C:C1:DE:66:13:DE	
kumquats	HP-xw4600-Core2Quad-SAS	4x2.83G	8Gb	Win7		general	356-malaiya	2UA9510WGJ	00:26:55:46:2d:09
eggplant	HP-xw6600-Xeon5450-SAS	8x3.0G	16Gb	Linux(Fedora)	r/t	475-machine-rm	2UA83910XM	00:1f:29:01:bb:11
endive		HP-xw6600-Xeon5450-SAS	8x3.0G	16Gb	Linux(Fedora)	r/t	475-machine-rm	2UA83910XB	00:1f:29:01:ab:bc
fennel		HP-xw6600-Xeon5450-SAS	8x3.0G	16Gb	Linux(Fedora)	r/t	475-machine-rm	2UA83910WZ	00:1f:29:01:bb:21
garlic		HP-xw6600-Xeon5450-SAS	8x3.0G	16Gb	Linux(Fedora)	r/t	475-machine-rm	2UA83910WS	00:1f:29:01:ab:c3
gourd		HP-xw6600-Xeon5450-SAS	8x3.0G	16Gb	Linux(Fedora)	r/t	475-machine-rm	2UA83910Y0	00:1f:29:01:bb:24
horseradish	HP-xw6600-Xeon5450-SAS	8x3.0G	16Gb	Linux(Fedora)	r/t	475-machine-rm	2UA83910WG	00:1f:29:01:bb:5b
kale		HP-xw6600-Xeon5450-SAS	8x3.0G	16Gb	Linux(Fedora)	r/t	475-machine-rm	2UA83910XV	00:1f:29:01:ab:2c
kelp		HP-xw6600-Xeon5450-SAS	8x3.0G	16Gb	Linux(Fedora)	r/t	475-machine-rm	2UA83910WN	00:1f:29:01:ab:fa
leek		HP-xw6600-Xeon5450-SAS	8x3.0G	16Gb	Linux(Fedora)	r/t	475-machine-rm	2UA83910XS	00:1f:29:01:bb:0e
lettuce		HP-xw6600-Xeon5450-SAS	8x3.0G	16Gb	Linux(Fedora)	r/t	475-machine-rm	2UA83910X3	00:1f:29:01:bb:2a
mushroom	HP-xw6600-Xeon5450-SAS	8x3.0G	16Gb	Linux(Fedora)	r/t	475-machine-rm	2UA83910XC	00:1f:29:01:bb:1a
okra		HP-xw6600-Xeon5450-SAS	8x3.0G	16Gb	Linux(Fedora)	r/t	475-machine-rm	2UA83910XL	00:1f:29:01:ab:b0
onion		HP-xw6600-Xeon5450-SAS	8x3.0G	16Gb	Linux(Fedora)	r/t	475-machine-rm	2UA83910X4	00:1f:29:01:bb:33
parsley		HP-xw6600-Xeon5450-SAS	8x3.0G	16Gb	Linux(Fedora)	r/t	475-machine-rm	2UA83910WL	00:1f:29:01:bb:65
parsnip		HP-xw6600-Xeon5450-SAS	8x3.0G	16Gb	Linux(Fedora)	r/t	475-machine-rm	2UA83910X6	00:1f:29:01:bb:1d
pea		HP-xw6600-Xeon5450-SAS	8x3.0G	16Gb	Linux(Fedora)	r/t	475-machine-rm	2UA83910WR	00:1f:29:01:ab:fe
pepper		HP-xw6600-Xeon5450-SAS	8x3.0G	16Gb	Linux(Fedora)	r/t	475-machine-rm	2UA83910XT	00:1f:29:01:bb:1b
potato		HP-xw6600-Xeon5450-SAS	8x3.0G	16Gb	Linux(Fedora)	r/t	475-machine-rm	2UA83910XY	00:1f:29:01:bb:1c
pumpkin		HP-xw6600-Xeon5450-SAS	8x3.0G	16Gb	Linux(Fedora)	r/t	475-machine-rm	2UA83910XG	00:1f:29:01:bb:16
radish		HP-xw6600-Xeon5450-SAS	8x3.0G	16Gb	Linux(Fedora)	r/t	475-machine-rm	2UA83910Y1	00:1f:29:01:bb:20
rhubarb		HP-xw6600-Xeon5450-SAS	8x3.0G	16Gb	Linux(Fedora)	r/t	475-machine-rm	2UA83910XW	00:1f:29:01:bb:0d
romanesco	HP-xw6600-Xeon5450-SAS	8x3.0G	16Gb	Linux(Fedora)	r/t	475-machine-rm	2UA83910XK	00:1f:29:01:bb:15
rutabaga	HP-xw6600-Xeon5450-SAS	8x3.0G	16Gb	Linux(Fedora)	r/t	475-machine-rm	2UA83910X7	00:1f:29:01:bb:18
shallot		HP-xw6600-Xeon5450-SAS	8x3.0G	16Gb	Linux(Fedora)	r/t	475-machine-rm	2UA83910WP	00:1f:29:01:ab:f8
spinach		HP-xw6600-Xeon5450-SAS	8x3.0G	16Gb	Linux(Fedora)	r/t	475-machine-rm	2UA83910XJ	00:1f:29:01:ab:a7
squash		HP-xw6600-Xeon5450-SAS	8x3.0G	16Gb	Linux(Fedora)	r/t	475-machine-rm	2UA83910XH	00:1f:29:01:ab:35
tomatillo	HP-xw6600-Xeon5450-SAS	8x3.0G	16Gb	Linux(Fedora)	r/t	475-machine-rm	2UA83910WT	00:1f:29:01:bb:28
tomato		HP-xw6600-Xeon5450-SAS	8x3.0G	16Gb	Linux(Fedora)	r/t	475-machine-rm	2UA83910X1	00:1f:29:01:ab:ff
turnip		HP-xw6600-Xeon5450-SAS	8x3.0G	16Gb	Linux(Fedora)	r/t	475-machine-rm	2UA83910WX	00:1f:29:01:ab:e7
wasabi		HP-xw6600-Xeon5450-SAS	8x3.0G	16Gb	Linux(Fedora)	r/t	475-machine-rm	2UA83910XN	00:1f:29:01:bb:06
yam		HP-xw6600-Xeon5450-SAS	8x3.0G	16Gb	Linux(Fedora)	general	475-machine-rm	2UA83910XR	00:1f:29:01:bb:12
zucchini	HP-xw6600-Xeon5450-SAS	8x3.0G	16Gb	Linux(Fedora)	general	475-machine-rm	2UA83910WH	00:1f:29:01:bb:61
uncompahgre	HP-Z210-XeonE3-1230	4x3.2G	8Gb	Linux(Fedora)	r/t	445		2UA12822SB	2c:27:d7:49:84:84
artichoke	HP-xw6600-Xeon5450-SAS	8x3.0G	16Gb	Win10		r/t	240-Wakefield	2UA83910X8	00:1F:29:01:BB:13
figs		HP-Z200-Xeon3450-SAS	4x2.66G	8Gb	Linux(Fedora)	general	252		2UA0330TMT	1c:c1:de:66:13:eb
grapes		HP-Z200-Xeon3450-SAS	4x2.66G	8Gb	Linux(Fedora)	general	252		2UA0330TMZ	1c:c1:de:66:d2:e0

machines.py
#!/usr/bin/python
import re
import itertools

def mysplit(line):
    groups = itertools.groupby(line, lambda k: k == '\t')
    runs = ["".join(vals) for k,vals in groups]
    fields = [run for run in runs if run[0] != '\t']
    return fields

machines = []
for lnum, line in enumerate(open("machines.2")):
    if lnum > 1:
        fields = mysplit(line.strip()) # re.split(r"\s+", line.strip())
        if len(fields) >= 7:
            machines.append(fields)


#print( "found", len(machines), "machines")
# NAME		TYPE			CPU	MEM	OS		USE	LOCATION	S/N		MAC ADDRESS
#execfile("usedmachines.py")
NAME = 0
TYPE = 1
CPU = 2
MEM = 3
OS = 4
USE = 5
LOCATION = 6
sn = 7


labs = ['478-storage',
        '475-machine-rm',
        '225-interac-lab',
        '215-interac-lab',
        '120-unix-lab']

morelabs = [    
'478-systems',
'325-hpc-lab',
'215-interac-lab',
'225-interac-lab',
# '315-netsec-lab',
# '478-storage',
# '405-storage',
'120-unix-lab',
'110-open-lab',
'475-machine-rm',
]

STRs = {
    "XeonE5-2650v2" : 1619,
    "XeonE3-1230" : 1823,
    "Xeon3450" : 1168,
    "Xeon5450" : 1274,
    "Xeon5450-SAS" : 1274,
    "X4100-amd254" : 889,
    "AMD-6386SE" : 988,
    "XeonE7-4830" : 1700,
    "AMD-6220" : 1062,
    "XeonE3-1231v3" : 2170,
    "XeonE5-1650v3" : 2116,
    "HP-DL160-G5" : 1149
}    


def get_STR(machine):
    typestring = machine[TYPE]
    "Get the single thread rating as an int"
    for k,v in STRs.items():
        if typestring[-len(k):] == k: return v
    raise Exception("STR not found", str(machine))


CPUs = 0
inset = 0
#goodmachines = set([ 'HP-xw6600-Xeon5450-SAS', 'HP-Z210-XeonE3-1230', 'HP-Z220-XeonE3-12230', 'HP-z420-XeonE5-2650v2'])
if __name__ == '__main':
    for machine in machines:
        if machine[OS] == 'Linux(Fedora)' and machine[NAME][0:len('lattice')] != 'lattice' and  machine[LOCATION] in morelabs:
        #print(machine[CPU], "\t\t", machine[MEM], "\t\t", machine[TYPE], get_STR(machine)) #machine[TYPE],
            print(machine[NAME])
            inset += 1
#     #    print machine[CPU]

#     if machine[NAME] in usedmachines: 
#         touse = int(machine[CPU][0]) - 1
#         inset += touse
#         print "    '" + machine[NAME] + "' : " + str(touse) + ","
#         #print machine[TYPE], machine[CPU], machine[MEM]

            mobj = re.search(r"^(\d+)x", machine[CPU])
            if mobj:
                CPUs += int(mobj.group(1))
#         #if machine[OS] == 'Linux(64)' and machine[USE] == 'general' and "(" not in  machine[NAME] and machine[LOCATION] in labs and machine[TYPE] == 'HP-xw4600-Core2Duo-SATA':# and machine[TYPE] in goodmachines:
#          #print  "    '" + machine[TYPE] + "' : 2," # machine[NAME], "\t\t\t",
#     #     print machine
# #        print machine[LOCATION]




#print( "all available CPUs:", CPUs)
#print( "in set", inset)
